steppers/ldo/42sth40-1684ac/2209/24v-0.4a-z.cfg

```diff
@@ -0,0 +1,9 @@
+[tmc2209 stepper_z]
+uart_pin: board_uart_pin
+uart_address: 0
+sense_resistor: 0.30
```

Comment:
These are board specific. Change uart_pin to `z_uart_pin` and move uart_address and sense_resistor to the prusa_buddy quirks filed i mentioned in another comment.

Reply:
Done